cookbooks/aws-parallelcluster-tests/recipes/tear_down.rb

```diff
@@ -1,3 +1,5 @@
 node.default['cluster']['node_virtualenv_path'] = node_virtualenv_path
 node.default['cluster']['cookbook_virtualenv_path'] = cookbook_virtualenv_path
 node_attributes 'Save values for InSpec tests'
+
+include_recipe "::unmount_home"
```

Comment:
Does it work locally?

Reply:
We're already using this approach in other recipes, e.g.:
``` 
include_recipe "::awsbatch_virtualenv"
```